nf_core/modules/test_yml_builder.py

```diff
@@ -74,7 +78,7 @@ def check_inputs(self):
 
         # Get the tool name if not specified
         if self.module_name is None:
-            modules_repo = ModulesRepo()
+            modules_repo = ModulesRepo(remote_url=self.remote_url, branch=self.branch)
             self.module_name = questionary.autocomplete(
                 "Tool name:",
                 choices=modules_repo.get_avail_components(self.component_type),
```

Comment:
I'm wondering if we can remove the use of the remote and just use this method from `ComponentsCommand`?

```suggestion
            self.module_name = questionary.autocomplete(
                "Tool name:",
                choices=self.components_from_repo(self.component_type),
```

Reply:
If this works we need to apply the same code into `subworkflows/test_yml_builder.py`